bindings/python/module.cpp

```diff
@@ -15,31 +15,30 @@
 // <http://www.gnu.org/licenses/>.
 //
 
-#include "tsid/bindings/python/fwd.hpp"
 #include <eigenpy/geometry.hpp>
 
-#include "tsid/bindings/python/robots/expose-robots.hpp"
 #include "tsid/bindings/python/constraint/expose-constraints.hpp"
 #include "tsid/bindings/python/contacts/expose-contact.hpp"
-#include "tsid/bindings/python/trajectories/expose-trajectories.hpp"
-#include "tsid/bindings/python/tasks/expose-tasks.hpp"
-#include "tsid/bindings/python/solvers/expose-solvers.hpp"
 #include "tsid/bindings/python/formulations/expose-formulations.hpp"
+#include "tsid/bindings/python/fwd.hpp"
```

Comment:
Are we sure that changing the order of the include isn't gonna affect the compilation?

Reply:
Good point I forgot to put the flag to avoid sorting the includes. I will do it and will check more carefully the CI tests. I got trapped in thinking that the CI was failing because of osqp (first CI failure on this PR https://github.com/stack-of-tasks/tsid/actions/runs/4126738738/jobs/7129052586).